.devcontainer/post-create.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+if [[ "${CODESPACES}" == true ]]; then
+  echo "Fixing permissions of /tmp for GitHub Codespaces..." >&2
+  sudo chmod 1777 /tmp
+fi
+
+set -x
+
+git config --global --add safe.directory "${PWD}"
+git config --global --add safe.directory "${PWD}/deps/renovate"
+
+git submodule update --init
+
+exec make
```

Comment:
we should run that as Ubuntu user, so the safe directory is no longer needed 

why the exec before make?

Reply:
> we should run that as Ubuntu user, so the safe directory is no longer needed

Good point.

> why the exec before make?

To delegate the execution and signal handling to the `make` command (it replaces Bash's process with Make's process). I believe it's a good practice for scripts that aims to prepare the execution of another process, especially for processes that takes significant time to execute.